### ISSUE TITLE
`Connect` button in server-selection dialog should respond to tabbable keyboard focus and enter events

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#40473]
### Description

`Connect` button in server-selection dialog should respond to tabbable keyboard focus and enter events

Thanks to @rebron for the find

### Steps to reproduce

1. install `1.70.77`
2. launch Brave
3. load `account.bravesoftware.com`
4. purchase and configure `Brave VPN`
5. click on `VPN` in the browser toolbar
6. 

### Actual result

The whole row is the tabbable element

![image](https://github.com/user-attachments/assets/182199bb-a6f7-40de-b7af-6cd0c58cc1f6)

### Expected result

The `Connect` button needs to be discretely tabbable (and should connect by pressing `enter`)

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

```
Brave | 1.70.77 Chromium: 128.0.6613.27 (Official Build) nightly (64-bit)
-- | --
Revision | 1c08cc9a735d56c81b9af5c80d3b136ed0ec15ff
OS | Windows 10 Version 22H2 (Build 19045.4780)
```

### Channel information

- [ ] release (stable)
- [ ] beta
- [X] nightly

### Reproducibility

- [X] with Brave Shields disabled
- [X] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

/cc @rebron @bsclifton @simonhong @brave/qa-team

**Comment by simonhong** [comment]
`Connect` gets focus by tab again after country row focused. Then, we can ask connect by `enter`.
Maybe you want to accept `enter` for connect when country row focused?
I did it intentionaly. Soon, we'll show server list per country. At that time, We could use `enter` for country row to unfold to show available server list.

**Comment by rebron** [comment]
@simonhong You're right, tab, tab does select `connect` and `enter` connects it. I think The second tab for connect button selection doesn't look highlighted/selected. The selection of `connect` button looks too subtle.

cc: @aguscruiz

**Comment by aguscruiz** [comment]
We have an issue in Leo to update the focus state of the filled buttons that hopefully will fix this
https://github.com/brave/leo/issues/730
